bower.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "ember-z-schema",
   "dependencies": {
-    "ember": "~2.5.0",
+    "ember": "2.6.0",
```

Comment:
Probably doesn't matter b/c the `bower.json` from an addon is only used in development, but pinning the version in an addon seems like the wrong thing to do.